src/clients/authorization/AuthorizationEvaluate.tsx

```diff
@@ -390,35 +434,34 @@ export const AuthorizationEvaluate = ({
               fieldId={name!}
             >
               <AttributeInput
-                selectableValues={defaultContextAttributes.map(
-                  (item) => item.name
-                )}
+                selectableValues={defaultContextAttributes}
                 isKeySelectable
+                reset={() => reset()}
                 name="context"
               />
             </FormGroup>
           </ExpandableSection>
-          <ActionGroup>
-            <Button data-testid="authorization-eval" type="submit">
-              {t("evaluate")}
-            </Button>
-            <Button
-              data-testid="authorization-revert"
-              variant="link"
-              onClick={reset}
-            >
-              {t("common:revert")}
-            </Button>
-            <Button
-              data-testid="authorization-revert"
-              variant="primary"
-              onClick={reset}
-              isDisabled
-            >
-              {t("lastEvaluation")}
-            </Button>
-          </ActionGroup>
         </FormAccess>
+        <ActionGroup>
+          <Button data-testid="authorization-eval" onClick={() => evaluate()}>
+            {t("evaluate")}
+          </Button>
+          <Button
+            data-testid="authorization-revert"
+            variant="link"
+            onClick={() => reset()}
+          >
+            {t("common:revert")}
+          </Button>
+          <Button
+            data-testid="authorization-revert"
+            variant="primary"
+            onClick={() => reset}
```

Comment:
You probably want to call the `reset()` function here.
```suggestion
            onClick={() => reset()}
```